tests/Microsoft.Identity.Test.Unit.net45/WebUITests/HttpResponseParserTest.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#if NET_CORE
+
+using System;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.Platforms.netcore.OsBrowser;
+using Microsoft.Identity.Test.Common.Core.Helpers;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NSubstitute;
+
+namespace Microsoft.Identity.Test.Unit.WebUITests
+{
+    [TestClass]
+    public class HttpResponseParserTest
+    {
+        [TestMethod]
+        public void HappyPath()
```

Comment:
nit: more meaningful names for the test methods (this and also the other new test file)?